torch/_torch_docs.py

```diff
@@ -4272,6 +4272,59 @@ def merge_dicts(*dicts):
     {out}
 """.format(**common_args))
 
+add_docstr(torch.xlogy,
+           r"""
+xlogy(x, y) -> Tensor
+
+Compute x*log(y) so that the result is 0 if x = 0.
+Similar to SciPy's `scipy.special.xlogy`.
+
+.. math::
+    \text{out}_i = x_i * \log{{y_i}}
+
+Args:
+    x (tensor): Multiplier
+    y (float or tensor): Argument
+
+.. note::
+    NaN in :attr:`y` is propagated even if value of :attr:`x` is 0.
+
+Example::
+
+    >>> x = torch.zeros(3,)
+    >>> y = torch.tensor([1, float('inf'), float('nan')])
+    >>> torch.xlogy(x, y)
+    tensor([0., 0., nan])
+    >>> x = torch.tensor([1, 2, 3])
+    >>> y = torch.tensor([3, 2, 1])
+    >>> torch.xlogy(x, y)
+    tensor([1.0986, 1.3863, 0.0000])
+    >>> torch.xlogy(x, 4)
+    tensor([1.3863, 2.7726, 4.1589])
+
+
+.. function:: xlogy(x, y) -> Tensor
```

Comment:
I don't think this needs a second entry?

Reply:
First entry covers two variants,
x -> tensor, y -> float, tensor

This entry covers,
x -> float, y -> tensor.

This is similar to `torch.pow`'s doc.
https://github.com/pytorch/pytorch/blob/a39398b9e5d528e4a6ca293f1703833932f0d9b2/torch/_torch_docs.py#L6403